algorithms/per/ddpg_agent.py

```diff
@@ -13,12 +13,12 @@
 
 from algorithms.common.buffer.priortized_replay_buffer import PrioritizedReplayBuffer
 import algorithms.common.helper_functions as common_utils
-from algorithms.ddpg.agent import Agent as DDPGAgent
+from algorithms.ddpg.agent import DDPGAgent
 
 device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
 
 
-class Agent(DDPGAgent):
+class DDPGPERAgent(DDPGAgent):
```

Comment:
Why not PERDDPG?
Think of the example file (per-ddpg.py) and other similar cases, e.g. bc.

Reply:
Changed DDPGPER to PERDDPG